olive/passes/vitis_ai/__init__.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Do you still need __init__.py ?

Reply:
No, files in /psses/vitis_ai/ has been moved to /passes/onnx, this file is OK to remove. Thanks very much for your review!